web/src/stream_popover.ts

```diff
@@ -471,7 +472,10 @@ export async function build_move_topic_to_stream_popover(
     }
 
     function update_submit_button_disabled_state(select_stream_id: number): void {
-        const {current_stream_id, new_topic_name, old_topic_name} = get_params_from_form();
+        const params = get_params_from_form();
+        const current_stream_id = params.current_stream_id;
+        const new_topic_name = params.new_topic_name?.trim();
+        const old_topic_name = params.old_topic_name?.trim();
```

Comment:
`old_topic_name` will always be present in `params` as seen from `params_schema` defined above this code, so it can just be `params.old_topic_name.trim()`.

Reply:
done!